src/Components/Main/Main.css

```diff
@@ -0,0 +1,12 @@
+/* background-img {
+  background-repeat: cover;
```

Comment:
Great job on getting the background in here. If you wanted to have just the single img without repeating you could try:
background-repeat: no-repeat;